theme.scss

```diff
@@ -59,11 +59,11 @@ $material-theme: mat-light-theme($material-theme-primary, $material-theme-accent
                 secondary-text: map-get($blui-gray, 500),
                 hint-text: map-get($blui-gray, 500),
                 base: map-get($blui-darkBlack, 900),
-                divider: rgba(66, 78, 84, 0.12),
+                divider: rgba(map-get($blui-black, 500), .12),
                 dividers: $dark-dividers,
-                disabled: rgba(66, 78, 84, 0.3),
+                disabled: rgba(map-get($blui-black, 500), .03),
                 disabled-button: rgba(black, 0.26),
-                disabled-text: rgba(66, 78, 84, 0.3),
```

Comment:
.3 opacity, not .03

Thanks for going through the themes and finding all these missed palette refs.

Reply:
Sure let me update.

Reply:
Hi @emclaug2 , please review.